proxy/deploy.go

```diff
@@ -79,14 +95,14 @@ func DeployFunction(fprocess string, gateway string, functionName string, image
 	SetAuth(request, gateway)
 	if err != nil {
 		fmt.Println(err)
-		return
+		return -1
```

Comment:
Not sure if this matters, but the usage shows that you're returning a status code (in the function call, not the signature), so should this return 500 (`http.StatusInternalServerError`) instead?

Same comment for line 105 with the other `return -1`

Purely a nit-pick thing, just thinking about consistency for down the road

Reply:
Yes what does -1 mean?